run-maintain.js

```diff
@@ -0,0 +1,3 @@
+#!/usr/bin/env node
+const { Maintain } = require('./maintain.js')
+Maintain()
```

Comment:
at this point, you know you're running from the cli

```
Maintain({...config...})
```

```
config = {
  cli: true
}
```

also a good place to get `process.argv`

maintain.js should just assume it's pure code